src/components/MovieCard/MovieCard.scss

```diff
@@ -1 +1,3 @@
-// 
+.card-image img {
```

Comment:
This rule doesn't work because you do not use this class name anymore.
Just rename it:
```
.image img {
```
to apply for this element:
![image](https://github.com/mate-academy/react_movies-list-js/assets/25587299/1b391edc-3a5b-417f-a7c6-15eb87739703)